### PR TITLE
[DOCFIX] Tweak yarn ec2 docs

### PR DESCRIPTION
**Pull request by aaudiber**
- A couple wording changes
- The master hostname is set deterministically, so we no longer need to search for it in the "Test Alluxio" section.

### CODE REVIEW

File: docs/en/Running-Alluxio-on-EC2-Yarn.md [hunk 1]
```diff
@@ -121,13 +121,13 @@ Note that adding `-DskipTests -Dfindbugs.skip -Dmaven.javadoc.skip -Dcheckstyle.
 but it makes the build run significantly faster.
 
 To customize Alluxio master and worker with specific properties (e.g., tiered storage setup on each
-worker), one can refer to [Configuration settings](Configuration-Settings.html) for more
-information. To ensure your configuration can be read by both the ApplicationMaster and Alluxio
-master/workers, put `alluxio-site.properties` under `~/.alluxio/` on each EC2 machine.
+worker), see [Configuration settings](Configuration-Settings.html). To ensure your configuration can be
+read by both the ApplicationMaster and Alluxio master/workers, put `alluxio-site.properties` under
+`~/.alluxio/` on each EC2 machine.
```
Comment (calvinjia) [inline docs/en/Running-Alluxio-on-EC2-Yarn.md]: Does this need to be in the home folder of the user the Alluxio process / AM is running under?

Comment (aaudiber) [inline docs/en/Running-Alluxio-on-EC2-Yarn.md]: @apc999

Comment (aaudiber) [inline docs/en/Running-Alluxio-on-EC2-Yarn.md]: updated

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**